blocks/library/image/index.js

```diff
@@ -50,7 +50,7 @@ registerBlock( 'core/image', {
 		{
 			icon: 'align-center',
 			title: wp.i18n.__( 'Align center' ),
-			isActive: ( { align } ) => 'center' === align,
+			isActive: ( { align } ) => ! align || 'center' === align,
```

Comment:
It appears the same in the editor, but different on the front-end.

none|center
---|---
![none](https://cloud.githubusercontent.com/assets/1779930/26258701/cabc7324-3c93-11e7-8800-c14b46732e17.png)|![center](https://cloud.githubusercontent.com/assets/1779930/26258711/db13c9a2-3c93-11e7-98c1-63a6083d600f.png)

We should probably make this distinction more obvious in the editor though.